### PR TITLE
Logging History service - 62

### PR DESCRIPTION
**Pull request by sajaadem**
## Description
Added the Logging History service

## Pre-review checks

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/UserSpecificEventStatus.java
Comment (Bailonis) [inline demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/UserSpecificEventStatus.java:1]: Pls change line 29 from: Done to DONE

Comment (sajaadem) [reply demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/entities/enums/UserSpecificEventStatus.java:1]: Done

**Review by Bailonis — APPROVED** [review]
All good from the team perspective.

**Comment by carslen** [comment]
There are errors in build-backend workflow for this PR:
https://github.com/eclipse-tractusx/demand-capacity-mgmt/actions/runs/6336887120/job/17387509240?pr=30

**Comment by sajaadem** [comment]
> There are errors in build-backend workflow for this PR: https://github.com/eclipse-tractusx/demand-capacity-mgmt/actions/runs/6336887120/job/17387509240?pr=30

@carslen Yeah, The rebase caused this issue today. Will work on resolving it.

**Comment by carslen** [comment]
please resolve conflicts

**Comment by carslen** [comment]
please take also care of the [failed check](https://github.com/eclipse-tractusx/demand-capacity-mgmt/actions/runs/6419805578/job/17596441131?pr=30).

**Comment by Bailonis** [comment]
@carslen resolved :)

**Pull request merged**